lib/component.js

```diff
@@ -70,11 +78,15 @@
     },
     // Sets `this.el` and `this.$el` when the component mounts.
     componentDidMount: function () {
-      this.setElement(React.findDOMNode(this));
+      if (isReactNative === false) {
+        this.setElement(React.findDOMNode(this));
+      }
     },
     // Sets `this.el` and `this.$el` when the component updates.
     componentDidUpdate: function () {
-      this.setElement(React.findDOMNode(this));
```

Comment:
Maybe something like `if (React.findDOMNode)` is better to avoid having to set an extra flag for it (`isReactNative`)?